neo/Consensus/ConsensusService.cs

```diff
@@ -220,17 +220,33 @@ private void OnChangeViewReceived(ConsensusPayload payload, ChangeView message)
 
         private void OnCommitReceived(ConsensusPayload payload, Commit commit)
         {
-            if (context.CommitPayloads[payload.ValidatorIndex] != null) return;
-            Log($"{nameof(OnCommitReceived)}: height={payload.BlockIndex} view={commit.ViewNumber} index={payload.ValidatorIndex}");
-            byte[] hashData = context.MakeHeader()?.GetHashData();
-            if (hashData == null)
+            ref ConsensusPayload existingCommitPayload = ref context.CommitPayloads[payload.ValidatorIndex];
```

Comment:
This is funny...aehuahuea Nice way to make it more readable.

Reply:
Thanks.